modules/ROOT/pages/resource-management.adoc

```diff
@@ -30,6 +30,12 @@ contentType=layout
 
 If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.
 
+[NOTE]
+
+====
+To insert line breaks in a page.properties file, end the line that preceeds the line break with a "\".
```

Comment:
Typo + Remove the double quote: they must be not added in the file
```suggestion
To insert line breaks in a page.properties file, end the line that precedes the line break with a `\`.
```

Reply:
Current rendering
![image](https://user-images.githubusercontent.com/27200110/142221764-797ef3c3-0931-4edb-af7b-3143753fc6c6.png)